### PR TITLE
[Dubbo-Optimization] Remove double loop

### PR DESCRIPTION
**Pull request by fzhyzamt**
## What is the purpose of the change

Optimize performance

## Brief changelog

Use `Map` to replace redundant traversal

## Verifying this change

Run unit test

Follow this checklist to help us incorporate your contribution quickly and easily:

- [ ] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes 
- [X] one PR resolves one issue.
- [X] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [X] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [X] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [X] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by AlbumenJ** [comment]
@fzhyzamt hi, thanks for your contribution

pls resolve conflicts with the latest master branch

File: dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/Wrapper.java [hunk 1]
```diff
@@ -1,453 +1,453 @@
...
-     *
-     * @param instance instance.
-     * @param mn       method name.
-     * @param types
-     * @param args     argument array.
-     * @return return value.
-     */
-    abstract public Object invokeMethod(Object instance, String mn, Class<?>[] types, Object[] args) throws NoSuchMethodException, InvocationTargetException;
-}
+/*
```
Comment (AlbumenJ) [inline dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/Wrapper.java:1]: Please recover the file end-line format of this file.

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/Wrapper.java (hunk 1 above)
Comment (fzhyzamt) [reply dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/Wrapper.java:1]: Strange, github will automatically add a empty line

**Comment by fzhyzamt** [comment]
@AlbumenJ This request contains a lot of redundant information, I created a clean request: https://github.com/apache/dubbo/pull/7493

**Pull request closed without merging**